src/java/org/apache/cassandra/repair/autorepair/RepairTokenRangeSplitter.java

```diff
@@ -687,8 +689,6 @@ static SizeEstimate getSizesForRangeOfSSTables(AutoRepairConfig.RepairType repai
 
                 long sstableSize = reader.bytesOnDisk();
                 totalBytes += sstableSize;
-                // TODO since reading the index file anyway may be able to get more accurate ratio for partition count,
```

Comment:
don't think we will actually change this, was just a consideration, so removing.